### ISSUE TITLE
Functions HOL may contain an error

### ISSUE THREAD

**Issue by dgkanatsios** [Microsoft/TechnicalCommunityContent#18]
Specifically, this line 
```
var results = await client.PostAsync("https://westus.api.cognitive.microsoft.com/vision/v1.0/analyze?visualFeatures=Adult", payload);
```

the location part of the URL depends on the Azure location where the key was created. Had two attendees in a lab today that created keys in West Europe and East US 2 locations, so the url should respectively be https://westeurope.api... and https://eastus2.api...
I would add a line to the text to pay attention to this.

**Comment by jeffprosise** [comment]
Good point, I plan to revise this lab to recommend that they select a certain region and warn them that they must change the URL if they don't.
